schemas/com.snowplowanalytics.snowplow/media_player_event_ad_quartile/jsonschema/1-0-0

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Media player event fired when a quartile of ad is reached after continuous ad playback at normal speed.",
```

Comment:
Which quartile (i.e., 25, 50, 75%) we should probably have this as a property if it doesn't exist somewhere else.

Reply:
It does exist as the `percentProgress` property in the ad entity schema. Still considering whether it should be here as wel...

Reply:
✅ I have added `percentProgress` to `ad_quartile_event`, `ad_skip_event`, `ad_click_event`, `ad_pause_event` and `ad_resume_event`. Have removed it from the `ad` entity.